xtrack/tracker.py

```diff
@@ -655,11 +657,13 @@ def _build_kernel(self, save_source_as, compile):
         src_lines.append(
             """
                         } //switch
+                    #ifndef SKIP_ACTIVE_CHECK_AND_SWAPS
```

Comment:
This is a bit dangerous (I put it in only to see)... How much do we lose if we don't skip it?

Reply:
Actually a lot... It totals a third of all the savings we make in this PR (on the same example simulation as above).

This is my summary:
| scenario | time/particle/turn [us] |
|---|---|
| everything on + faster Faddeeva | 54.78 (+37%) |
| everything in this PR on, no Faddeeva | 65.65 (+24%) |
| no `SKIP_ACTIVE_CHECK_AND_SWAPS`, ditto | 80.18 (+7%) |
| everything in this PR off, ditto | 86.58 |